### DISCUSSION TITLE
Document: use on Fedora Silverblue with Distrobox

### DISCUSSION THREAD

**Discussion by aral** [donniebreve/touchcursor-linux#71] [General]
1. Install per the instructions here from inside of Distrobox
2. Type `distrobox export --service touchcursor` from within Distrobox.

Then follow Distrobox’s instructions for the service name to use to start the service, etc.

(Thanks for making/sharing this; it’s very cool.)

**Comment by donniebreve** [comment]
Thanks @aral. I'm surprised you can run a service from within Distrobox. I shouldn't be... but I am! Distrobox is pretty great.

For # 1, you wrote "here" but didn't provide a link :smile: 

Maybe this helps @kadykov #68
